src/types.js

```diff
@@ -1,9 +1,19 @@
 import _ from 'lodash/fp'
 import * as F from 'futil-js'
 
-export let getTypeProp = _.curry((types, prop, node) =>
-  F.cascade([`${node.type}.${prop}`, `default.${prop}`], types)
+// Gets type a specific property from any of the places it might be - on the type in `types`, on default in `types`, or already on the node itself
```

Comment:
It might be better to split this line above into multiple lines. Editors don’t always wrap long lines.